joomla-framework/session
# [4.0] Pull request for session timer not working.

.

Pull Request for Issue #18921

### Summary of Changes
Validate session if the session isn't valid and if it expired then try to restart else destroy it

### Testing Instructions
Apply patch and do nothing for period of time you set in session timer (lifetime).

###Expected result

auto logout.

###Actual result

still active.
